includes/integrations/class-member-chi-membership-plugin-integrations.php

```diff
@@ -28,16 +28,28 @@ public function define_hooks() {
 	 *
 	 */
 	public function load_integrations() {
-		require_once dirname(__FILE__) . '/class-member-chi-membership-plugin-integration.php';
+
+		require_once dirname( __FILE__ ) . '/class-member-chi-membership-plugin-integration.php';
 		if ( class_exists( 'RCP_Levels' ) ) {
-			require_once dirname(__FILE__) . '/restrict-content-pro/class-member-chi-restrict-content-pro-integration.php';
+			require_once dirname( __FILE__ ) . '/restrict-content-pro/class-member-chi-restrict-content-pro-integration.php';
 			new Member_Chi_Restrict_Content_Pro_Integration();
 		}
 
 		if ( class_exists( 'WPComplete' ) ) {
-			require_once dirname(__FILE__) . '/wp-complete/class-member-chi-wp-complete-integration.php';
+			require_once dirname( __FILE__ ) . '/wp-complete/class-member-chi-wp-complete-integration.php';
 			new Member_Chi_Wp_Complete_Integration();
 		}
+
+		if ( is_woocommerce_active() && is_plugin_active( 'woocommerce-memberships/woocommerce-memberships.php' ) ) {
```

Comment:
good catch - the function is included in WC Memberships, but with a note saying not to rely on using it.